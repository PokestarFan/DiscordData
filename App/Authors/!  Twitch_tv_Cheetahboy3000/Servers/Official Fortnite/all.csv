ID,Channel,Content Length,Content
560642432505610260,fortnite-general,18,any youtubers here
560642582359834635,stream-video-promo,18,any youtubers here
560642909934977034,fortnite-general,31,this game is dead why am i here
560643430892699668,fortnite-general,56,<@!240671089544527873> u can only use it every six hours
560643499775492099,fortnite-general,25,dosent this count as spam
560643638032334871,fortnite-general,43,<@!137086233124798466> lot of spam going on
560643730030329876,fortnite-general,5,oh no
