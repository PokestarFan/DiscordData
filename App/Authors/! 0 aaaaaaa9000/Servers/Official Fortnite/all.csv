ID,Channel,Content Length,Content
558792322213609482,stw-trade,47,[H] Malachite [W] Lots and lots of brick [P] PC
