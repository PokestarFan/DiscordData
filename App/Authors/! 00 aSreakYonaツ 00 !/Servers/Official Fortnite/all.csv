ID,Channel,Content Length,Content
559031027239223296,fortnite-general,26,anyone in asia down to 1v1
