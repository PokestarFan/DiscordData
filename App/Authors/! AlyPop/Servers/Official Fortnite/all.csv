ID,Channel,Content Length,Content
558805625774145568,stream-video-promo,30,https://www.twitch.tv/alypop ðŸ˜®
