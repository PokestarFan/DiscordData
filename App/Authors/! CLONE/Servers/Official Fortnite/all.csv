ID,Channel,Content Length,Content
558860855391944750,fortnite-general,52,real og's will have 20 extra vbucks in their account
