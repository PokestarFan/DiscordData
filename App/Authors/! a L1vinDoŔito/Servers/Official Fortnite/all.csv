ID,Channel,Content Length,Content
560625274471841793,save-the-world,13,drum gun best
560629598682087447,fortnite-general,34,I personally thought its was trash
560629822280171520,fortnite-general,54,They shouldn't have removed the 0 health /shield thing
560629944464703518,fortnite-general,42,Alot of player relied on that extra health
560630656611385364,fortnite-general,86,https://tenor.com/view/partying-party-hard-dancing-awkward-awkward-dancing-gif-7830424
560630735753707521,fortnite-general,16,Update sucks doh
560630921641066496,fortnite-general,46,its so stupid <@!463480278686498828> ur stupid
560631006474797057,fortnite-general,22,this kid is and iddiot
560631156605976577,fortnite-general,3,yas
560631252454080532,fortnite-general,17,How is Ea support
560631474907381760,fortnite-general,3,lol
560631554003566596,fortnite-general,38,Apex is't that much play to win though
560631624866201642,fortnite-general,4,*pay
