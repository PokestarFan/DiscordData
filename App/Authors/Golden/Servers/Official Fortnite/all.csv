ID,Channel,Content Length,Content
560976830648287276,battle-royale,71,it took them forever to get magma in the shopNLi've been waiting so long
